test/RPCSession.cpp

```diff
@@ -32,6 +32,34 @@
 using namespace std;
 using namespace dev;
 
+#if defined(_WIN32)
```

Comment:
I don't know that it is worth us carrying this code around just for that one error message.

I assume this was useful to you while troubleshooting this implementation, but maybe doesn't have lasting value?